pkg/config/auth_test.go

```diff
@@ -121,29 +136,44 @@ func TestHookDiff(t *testing.T) {
 	t.Run("local and remote enabled", func(t *testing.T) {
 		c := newWithDefaults()
 		c.Hook = hook{
-			CustomAccessToken:           hookConfig{Enabled: true},
-			SendSMS:                     hookConfig{Enabled: true},
-			SendEmail:                   hookConfig{Enabled: true},
-			MFAVerificationAttempt:      hookConfig{Enabled: true},
-			PasswordVerificationAttempt: hookConfig{Enabled: true},
+			CustomAccessToken:           defaultHttpHookConfig(),
+			SendSMS:                     defaultHttpHookConfig(),
+			SendEmail:                   defaultHttpHookConfig(),
+			MFAVerificationAttempt:      defaultHttpHookConfig(),
+			PasswordVerificationAttempt: defaultPostgresHookConfig(),
```

Comment:
Let's not refactor the test helpers here. explicitly defining each hook is more easier to read. Also means we can set pg-function and http hook in one test